management-api-server/src/main/java/com/datastax/mgmtapi/resources/v1/NodeOpsResources.java

```diff
@@ -59,10 +65,30 @@ public Response decommission(@QueryParam(value="force")boolean force)
     )
     public Response rebuild(@QueryParam("src_dc") String srcDatacenter)
     {
-        return com.datastax.mgmtapi.resources.NodeOpsResources.handle(() ->
+        return handle(() ->
                 Response.accepted(
                         ResponseTools.getSingleRowStringResponse(app.dbUnixSocketFile, cqlService, "CALL NodeOps.rebuild(?)", srcDatacenter)
                 ).build());
     }
 
+    @GET
+    @Path("/schema/versions")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiResponse(responseCode = "200", description = "Gets the schema versions for each node. Useful for checking schema agreement")
```

Comment:
To make the OpenAPI spec generation accurate, you need to add an example content to `APIResponse`. It should probably be like this:
```java
  @ApiResponse(
      responseCode = "200",
      description = "Gets the schema versions for each node. Useful for checking schema agreement",
      content =
          @Content(
              mediaType = MediaType.APPLICATION_JSON,
              examples = @ExampleObject(value = "{2207c2a9-f598-3971-986b-2926e09e239d: [10.244.1.4, 10.244.2.3, 10.244.3.3]}")))
```

Once you have this, a `mvn compile` should update the openapi spec file too